### ISSUE TITLE
Remove from ORG

### ISSUE THREAD

**Issue by jkubon** [forcedotcom/CustomMetadataLoader#62]
Hello, how do I uninstall this from our Production ORG?  Are there steps that have automated the removal of the tool?  Thank you.

**Comment by rlancejohnson** [comment]
Did we ever get this figured out?

**Comment by jkubon** [comment]
Yes.  I was able to remove it from PROD.  Saved me for removing a slew of unmanaged APEX.

General process.  I took the Package.XML referenced here on GitHub.  Created a sandbox ORG from PROD and practiced with Workbench.  (See link below on steps how to do that.)  I had to iterate through somethings to make it work.

One thing in particular is the package XML provided uses a wildcard to delete all Labels (I think it was) and it cleared, of course, ALL the labels, including the ones from other efforts.  So ultimately to that out, looked for labels from the Namespace, and deleted them manually instead.

https://www.salesforceben.com/way-to-delete-apex-classes-from-production/

Good luck.

Joseph Kubon
SWBC – Director of Cloud Solutions
The Financial Institutions Group
210.503.4287
210.310.6147 cell

Confidentiality Notice | This e-mail message, including any attachments, is for the sole use of the intended recipient(s) and may contain confidential or proprietary information.  Any unauthorized review, use, disclosure or distribution is prohibited.  If you are not the intended recipient, immediately contact the sender by reply e-mail and destroy all copies of the original message.
